<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortCalcuDig</class>
 <widget class="QDialog" name="ShortCalcuDig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>309</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>短路计算</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="9" column="4">
    <widget class="QPushButton" name="pbn_Cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="pbn_Ok">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lab_Z">
     <property name="text">
      <string>Z(Ω),Phi(°):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>故障类型:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="cmb_ChGroup"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="cmb_FaultType"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>组号:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lne_Phi"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lab_RX">
     <property name="text">
      <string>R(Ω),X(Ω):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="rad_ZPhi">
     <property name="text">
      <string>Z,Phi</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="rad_RX">
     <property name="text">
      <string>R,X</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lne_Z"/>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lne_X"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>计算模型:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lne_R"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QComboBox" name="cmb_CacuMode"/>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="lne_LoadI"/>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>负荷电流(A):</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lab_DLI">
     <property name="text">
      <string>短路电流(A):</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>负荷功角(°):</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="lne_ShortI"/>
   </item>
   <item row="7" column="4">
    <widget class="QLineEdit" name="lne_LoadAngle"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lab_DLV">
     <property name="text">
      <string>短路电压(V):</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLineEdit" name="lne_ZSZL"/>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLineEdit" name="lne_ShortU"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lab_ZSZL">
     <property name="text">
      <string>|ZS/ZL|:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="4" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>零序补偿系数：</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>计算方式：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmb_CacuType"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lab_Para1">
        <property name="text">
         <string>幅值：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lne_CacuPara1"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lab_Para2">
        <property name="text">
         <string>相角：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lne_CacuPara2"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
